src/main/java/net/minestom/server/Viewable.java

```diff
@@ -101,6 +101,6 @@ default void sendPacketToViewersAndSelf(@NotNull ServerPacket packet) {
      * @return the audiences
      */
     default @NotNull Iterable<? extends Audience> getViewersAsAudiences() {
-        return this.getViewers();
+        return this.getViewers().unwrap().collect(Collectors.toList());
```

Comment:
I'm not sure about this, the only thing audience will ever do is directly send packets - is there a way we could access this directly without having to unwrap?

Reply:
`unwrap` means that there is no acquisition happening

Reply:
Yeah but there must be a better way than going from a collection to a stream and back to a list every time the use just wants to access thread safe methods like this. Could we use the collector view class for this?

Reply:
We could also use `#iterator` https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/stream/BaseStream.html#iterator()
it does however add an ugly warning. And I do not expect this to have a huge performance impact, could be wrong